packages/replay-internal/src/replay.ts

```diff
@@ -1046,11 +1046,23 @@ export class ReplayContainer implements ReplayContainerInterface {
    * are included in the replay event before it is finished and sent to Sentry.
    */
   private _addPerformanceEntries(): Promise<Array<AddEventResult | null>> {
-    const performanceEntries = createPerformanceEntries(this.performanceEntries).concat(this.replayPerformanceEntries);
+    let performanceEntries = createPerformanceEntries(this.performanceEntries).concat(this.replayPerformanceEntries);
 
     this.performanceEntries = [];
     this.replayPerformanceEntries = [];
 
+    // If we are manually starting, we want to ensure we only include performance entries
+    // that are after the initial timestamp, with 1s wiggle room.
+    // The reason for this is that we may have performance entries from the page load, but may decide to start
+    // the replay later on, in which case we do not want to include these entries.
+    // without this, manually started replays can have events long before the actual replay recording starts,
+    // which messes with the timeline etc.
+    if (this._requiresManualStart) {
+      // We leave 1s wiggle room to accomodate timing differences for "immedidate" manual starts
+      const initialTimestampInSeconds = this._context.initialTimestamp / 1000 - 1;
```

Comment:
the problem is that if you have code like this, I guess:

```js
Sentry.init({
  replaysErrorSampleRate: 0,
  replaysSessionSampleRate: 0, 
  integrations: [replay]
});

if (isImportantUser) {
  replay.start();
}
```

Like this kind of pattern, where users start manually but "immediately", we would otherwise probably miss many of the page load performance entries, as they are often correctly pre-dated to `Date.now()` 🤔 but no strong feelings, it is a bit "random". happy to generally remove the wiggle room!

Reply:
Ah didn't consider that... I think we keep it simple and remove the wiggle room, but don't feel strongly about this.